src/components/Toolbar/_mocks_/index.js

```diff
@@ -52,6 +52,7 @@ const applicationsToGenerate = [
     ]),
   },
   {
+    onClick: console.log,
```

Comment:
Did you mean to use this over an "action" in storybook?

Reply:
Good idea!